### ISSUE TITLE
[bug?] directives outside namespace will remove current usings

### ISSUE THREAD

**Issue by 304NotModified** [mattwhitfield/Unitverse#60]
Given this test class

```c#
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading;
using FluentAssertions;
using Moq;
using Xunit;

namespace MyNameSpace
{
    using MyNameSpace.Domain; // Note!

    public class MyTests{ 

....

}
```

1. Enable directives outside namespace.   ![image](https://user-images.githubusercontent.com/5808377/155562675-4270ee43-47b3-44e0-a4da-d59d1ce0a81c.png)
1. Generate a test methode for an existing class (and with exiting tests class)

Current result
1. Test will be added (nice!)
2. Almost all usings are removed! (not expected)

Result: 
```c#
using MyNameSpace.Domain; 
// Missing usings

namespace MyNameSpace
{
        public class MyTests{ 

        [Fact]
        public void CanCall() {... }


}
```

Note:
- when missing `using MyNameSpace.Domain; // Note!`, we have as a result 0 usings
- this isn't an issue when directives outside is `false`

**Comment by mattwhitfield** [comment]
Ok, this one I won't address as quick as usual as I am away for the next few days. But I will get on it.

**Comment by mattwhitfield** [comment]
Actually, scrub that. It's a fairly simple fix. But I am going to raise another item because I need to add tests for generation over the top of existing output, and that will take time.

**Comment by mattwhitfield** [comment]
Fixed by 0.103 - was a stupid mistake. Created #62 to try and stop me making silly mistakes 🤣

**Comment by 304NotModified** [comment]
Thanks again! Great work!
